### ISSUE TITLE
Inplace: add keyboard support

### ISSUE THREAD

**Issue by jepsar** [primefaces/primefaces#7398]
Output should be focusable and enter should trigger a click.

See also: https://stackoverflow.com/questions/67901875/how-to-tab-correctly-to-pinplace-component-in-jsf-so-screenreader-can-read-the
